core/migrations/0013_sepdata_needles_exchanged.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.13 on 2021-01-21 19:41
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('core', '0012_auto_20201216_0954'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='sepdata',
+            name='needles_exchanged',
```

Comment:
i think the name should be exchanged_for.

" the number of people that are involved with this exchange"

Reply:
@MikeyManoguerra Should this change happen on the frontend as well, or are you just concerned about how the data will appear in the database?

Reply:
front end too if you can, try and keep in I sync @JackRyan1989

Reply:
there should be three SEP fields
1. needles_in: the number of syringes they brought in to exchange
2. needles_out: the number of syringes PPP gave them 
3. exchanged_for: the number of people they are exchanging needles for